Pipfile

```diff
@@ -8,8 +8,9 @@ grpcio = "~=1.33.2"
 grpcio-tools = "~=1.33.2"
 requests = "~=2.25.0"
 python-json-logger = "~=0.1.11"
-cryptography = "~=3.2.1"
+cryptography = "~=3.3.1"
```

Comment:
cryptography dependency has just been updated to `3.4.4`, rebase to pick that change.